testsuite/features/init_clients/sle_buildhost.feature

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) 2016-2020 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Be able to bootstrap a Salt buildhost via the GUI
+
+@buildhost
+  Scenario: Create the bootstrap repository for a Salt client buildhost
+     Given I am authorized
+     And I create the "x86_64" bootstrap repository for "buildhost" on the server
+
+@buildhost
+  Scenario: Bootstrap a SLES buildhost
+     Given I am authorized
+     When I go to the bootstrapping page
+     Then I should see a "Bootstrap Minions" text
+     When I enter the hostname of "buildhost" as "hostname"
+     And I enter "22" as "port"
+     And I enter "root" as "user"
+     And I enter "linux" as "password"
+     And I select the hostname of "proxy" from "proxies"
+     And I click on "Bootstrap"
+     And I wait until I see "Successfully bootstrapped host!" text
+
+@buildhost
+  Scenario: Check the new bootstrapped buildhost in System Overview page
+    Given I am authorized
+    When I go to the minion onboarding page
+    Then I should see a "accepted" text
+    When I follow the left menu "Systems > Overview"
+    And I wait until I see the name of "buildhost", refreshing the page
+    And I wait until onboarding is completed for "buildhost"
+    Then the Salt master can reach "buildhost"
+
+@proxy
+@buildhost
+  Scenario: Check connection from buildhost to proxy
+    Given I am on the Systems overview page of this "buildhost"
+    When I follow "Details" in the content area
+    And I follow "Connection" in the content area
+    Then I should see "proxy" short hostname
+
+@proxy
+@buildhost
+  Scenario: Check registration on buildhost of minion
+    Given I am on the Systems overview page of this "proxy"
+    When I follow "Details" in the content area
+    And I follow "Proxy" in the content area
+    Then I should see "buildhost" hostname
+
+@buildhost
+  Scenario: Subscribe the SLES buildhost to a base channel
+    Given I am on the Systems overview page of this "buildhost"
+    When I follow "Software" in the content area
+    And I follow "Software Channels" in the content area
+    And I wait until I do not see "Loading..." text
+    And I check radio button "Test-Channel-x86_64"
+    And I wait until I do not see "Loading..." text
+    And I click on "Next"
+    Then I should see a "Confirm Software Channel Change" text
+    When I click on "Confirm"
+    Then I should see a "Changing the channels has been scheduled." text
+    And I wait until event "Subscribe channels scheduled by admin" is completed
+
+  # bsc#1085436 - Apache returns 403 Forbidden after a zypper refresh on minion
```

Comment:
This bug still applying?

Reply:
It's fixed.  What should I do? Just delete the comment or something else?

Reply:
Well, I'm not sure if the action behind it is made for that reason.
Check who implemented the step definition. Apparently it could be some relation. Maybe we can remove some hack or remove the scenario from that feature.